src/EditorFeatures/Core/Implementation/Classification/Semantic/ClassificationUtilities.cs

```diff
@@ -63,15 +78,17 @@ internal static class SemanticClassificationUtilities
             // We weren't able to use our specialized codepaths for semantic classifying. 
             // Fall back to classifying the full span that was asked for.
             await ClassifySpansAsync(
-                context, spanToTag, classificationService, typeMap, options, cancellationToken).ConfigureAwait(false);
+                context, document, spanToTag.SnapshotSpan, classificationService, typeMap, options, type, cancellationToken).ConfigureAwait(false);
         }
 
         private static async Task<bool> TryClassifyContainingMemberSpanAsync(
             TaggerContext<IClassificationTag> context,
-            DocumentSnapshotSpan spanToTag,
+            Document document,
+            SnapshotSpan snapshotSpan,
```

Comment:
i broke this into two parameters in some cases as DocumentSnapsthoSpan has a nullable Document, and that makes the NRT analysis annoying.